pvm.py

```diff
@@ -35,116 +43,230 @@ def _init_logger():
 	handler.setFormatter(formatter)
 	logger.addHandler(handler)
 
+# Log system initiated
 _init_logger()
 _logger = logging.getLogger("PVM")
 _logger.info("Logging system initiated in %s", LOG_PATH)
 
-media = ""
-
 '''
 UDP command example:
 /PVM HH MM SS <command> <value>
 '''
 
+# Parse UDP command
 def parse_commands(*args):
-	global media
-	global VIDEO_PATH
+	global media1
+	global media2
+	global VIDEO_PATH_ONE
+	global VIDEO_PATH_TWO
 	global IS_FILE_SET
-	
-	# Parse time and add 3 seconds
-	hh, mm, ss = args[1], args[2], args[3]
-	time_str = str(hh) + ":" + str(mm) + ":" + str(ss)
-	next_time = datetime.strptime(time_str, "%H:%M:%S") + timedelta(seconds=3)
+	global canPause
+	global canStart
+	global displayNum
 
 	# Get command
 	command = args[4]
 
```

Comment:
should you use the alternative `parse_value` function below

```suggestion
	command = args[4]
	# Get values
	values = parse_value(args[5])
```